lib/immobilienscout/version.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module Immobilienscout
-  VERSION = '0.0.6'.freeze
+  VERSION = '0.0.6'
```

Comment:
do we wanna bump the version? after all, changes were made :)

Reply:
Gonna happen later.